config.lua

```diff
@@ -0,0 +1,18 @@
+Config = {}
+
+Config['CreateApartmentFurnished'] = {
+    ['model'] = 'furnitured_midpart',
+    JSON = {
```

Comment:
You can convert all the coords to vectors, no need to decode what's already there